tests/test_header_keywords.py

```diff
@@ -6,6 +6,7 @@
 from astropy.io import fits
 import numpy as np
 import galsim
+import imsim
```

Comment:
I don't think this is the best way to make this module available.  It's not explicitly used in the code below, so I think it's confusing to have it imported here.   I think it makes more sense to add a `modules` entry in the `config` below, i.e.,
```
    config = {'modules': ['imsim'],
              'template': template,
```
This would be consistent with how that module would be made available in an actual yaml file.

Reply:
Done.